### DISCUSSION TITLE
Un-ignore "tests" folder

### DISCUSSION THREAD

**Discussion by mu88** [renovatebot/renovate#23631] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Hi,

I'm running Renovate Bot on my GitHub repo and was wondering why my project `Tests.csproj` is not recognized as NuGet dependency: https://github.com/mu88/ScreenshotCreator/issues/3

This my [Renovate config](https://github.com/mu88/ScreenshotCreator/blob/main/renovate.json):
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    "group:all",
    ":semanticCommitTypeAll(chore)",
    "schedule:earlyMondays"
  ]
}
```

After checking [the logs](https://developer.mend.io/github/mu88/ScreenshotCreator/-/job/0189a9e4-70df-794d-805b-bae983276460) and [the docs](https://docs.renovatebot.com/configuration-options/#ignorepaths), I'm pretty sure that it is related to the fact that the directory `tests` is excluded by default.

Now I'd like to learn what the best way is to un-ignore the tests folder so that my project gets recognized by Renovate.

Thank you

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: Found repo ignorePaths
{
  "ignorePaths": [
    "**/node_modules/**",
    "**/bower_components/**",
    "**/vendor/**",
    "**/examples/**",
    "**/__tests__/**",
    "**/test/**",
    "**/tests/**",
    "**/__fixtures__/**"
  ]
}
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by viceice** [comment]
https://docs.renovatebot.com/presets-default/#ignoremodulesandtests

ignore this preset and set your own `ignorePaths`

**Reply by mu88** [reply]
And how do I ignore this particular preset? Do I have to completely remove `"config:base"` and mimic all which is related to that?

**Reply by rarkins** [reply, marked as answer]
Use ignorePresets
